test/button_test.js

```diff
@@ -10,6 +10,100 @@ const Context = firefox.Context;
 // TODO create new profile per test?
 // then we can test with a clean profile every time
 
+async function urlBarTest(driver) {
```

Comment:
It looks like this, and most of the other tests you pulled out into separate methods, are only used in their original locations now. Do you still want to keep them separate?